receiver/expvarreceiver/README.md

```diff
@@ -1,12 +1,16 @@
 # Expvar Receiver
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [alpha]   |
+| Supported pipeline types | metrics   |
+| Distributions            | [contrib] |
```

Comment:
Does the contrib link still need added?

Reply:
ah yes, sorry, I missed that. I've added the links at the bottom.